source/templates/external_doc_template.html.erb

```diff
@@ -1,6 +1,6 @@
 ---
 layout: api_layout
```

Comment:
This is the only suggestion I'm hesitant about, as it involves a UI change. By removing the `api_layout`, we lose the menu that shows the imported docs, e.g.

<img width="321" alt="Screenshot 2020-12-29 at 16 56 42" src="https://user-images.githubusercontent.com/5111927/103300492-dc325280-49f6-11eb-98a1-4265e810f772.png">

I was hoping to tackle UI changes in #2912 instead. I experimented with moving these new menu items to the `app_layout` but it felt very cluttered, so this felt like a possible area of contention which I wanted to move to a subsequent PR rather than try to tack on here.

Reply:
Oh, interesting. I see the `application_layout` is currently used [here](https://github.com/alphagov/govuk-developer-docs/blob/c3499bb0ee87677efbe6af38b47b511fb8edbb72/source/templates/application_template.html.md.erb#L2). However, I'm confused about how the layout affects the menu, which also lead me to question: how will this PR affect the current "Apps" menu (the one which shows "Application ownership by team")?

(I think there's definitely scope for improvement in the current navigation: it's really unclear what the relationship between the left sidebar and the header is - they seem to be totally different navigation systems!)

Reply:
The layout affects the menu ('sidebar') [here](https://github.com/alphagov/govuk-developer-docs/blob/b081338b0a11cb561df200124e82e2533ed32728/source/layouts/application_layout.html.erb#L1-L24) - it sets the content for the sidebar. API layout [has its own sidebar](https://github.com/alphagov/govuk-developer-docs/blob/beade7407c23678a341374abcf1b35ab5d6c591d/source/layouts/api_layout.html.erb#L1-L16) which I've left relatively untouched for the purposes of just enabling 'consume docs' functionality across all repos. It therefore looks no different in this branch.

In a spike, I tried removing the API layout and moving the sidebar content into Application layout, but it gets very noisy. I'm taking a different approach in #2912.

Reply:
Oh, I see. Thanks for explaining. And I see the home page sidebar is [statically inconsistent](https://github.com/alphagov/govuk-developer-docs/blob/b081338b0a11cb561df200124e82e2533ed32728/data/dashboard.yml) with the header links. Lovely! I think I see where you're going in the follow-up PR - I'll be interested to see what you do with the sidebar!